.codecov.yml

```diff
@@ -14,11 +11,11 @@ coverage:
   status:
     changes: false
     patch:
-      default:
-        threshold: 0.01
-    project:
-      default:
-        threshold: 0.01
+      only_pulls: true
+      paths:
```

Comment:
According to https://docs.codecov.io/docs/commit-status#section-patch-status, we don't need to add specific directories. It should only look for the changes in PR.

We can also add directories to `ignore`: https://docs.codecov.io/docs/ignoring-paths